.github/workflows/rotki_nightly_builds.yml

```diff
@@ -205,12 +205,12 @@ jobs:
       - name: Notify On Failure
         if: ${{ failure() }}
         run: |
-          curl --request POST \
-            --url ${{ secrets.WEBHOOK }} \
-            --header 'Content-Type: application/json' \
-            --data '{
-          	"content": ":construction_site: **Github Actions:** ${{ github.job }} failed :x: \r\n\r\n  <@&735068019440615516> please have a look at [${{ github.run_id }}](https://github.com/rotki/rotki/actions/runs/${{ github.run_id }}) :cry:"
-          }'
+          $uri="${{ secrets.WEBHOOK }}"
+          $header = @{ "ContentType" = "application/json" }
+          $body = @{
+            "content" = ":construction_site: **Github Actions:** ${{ github.job }} failed :x: <@&735068019440615516> please have a look at [${{ github.run_id }}](https://github.com/rotki/rotki/actions/runs/${{ github.run_id }}) :cry:"
+          } | ConvertTo-Json
+          Invoke-RestMethod -Method Post -Uri $uri -Header $header -ContentType "application/json" -Body $body
```

Comment:
Also if you define contentType in the header shouldn't you avoid adding it again?

Reply:
You are probably right, after checking the documentation it seems that the parameter is also about the request body.
I just copied some examples and since it worked I didn't bother more.

It should be extraneous I will have to check first to make sure that it is indeed so.

Reply:
Ok, it is extraneous indeed so I removed it. I don't think it created any problem but it was an unnecessary duplication of the same thing

Reply:
So merge?